aten/src/ATen/core/dispatch/README.md

```diff
@@ -8,5 +8,5 @@ This folder contains the following files:
 - DispatchTable.h: Implementation of the actual dispatch mechanism. Hash table with kernels, lookup, ...
 - KernelCache.h: An interface operator kernels can use to inherit from if they need to keep around a cache between invocations
 - KernelFunction.h: The core interface (i.e. function pointer) for calling a kernel
-- OpSchemaRegistration.h: The mechanisms to register new operators with the c10 dispatcher
-- KernelRegistration.h: The mechanisms to register kernels with the c10 dispatcher
+- OpSchemaRegistration.h (deprecated): The mechanisms to register new operators with the c10 dispatcher
```

Comment:
shall we add new files to this list?

Reply:
This README collects the files in core/dispatch/, but the new files added in this diff are in a different directory.